runtime/src/lib.rs

```diff
@@ -960,6 +962,8 @@ impl pallet_file_bank::Config for Runtime {
 	type FindAuthor = pallet_session::FindAccountFromAuthorIndex<Self, Babe>;
 	type WeightInfo = pallet_file_bank::weights::SubstrateWeight<Runtime>;
 	type MinerControl = Sminer;
+	// Be aware of this: https://github.com/paritytech/polkadot/pull/3347
+	// Using this pallet as a randomness source is advisable primarily in low-security situations
```

Comment:
I replaced the source of randomness, choosing the source used by our current consensus:
https://github.com/CESSProject/cess/pull/88/files#diff-0ec06ea58bd455f09ce6b3bb4c2c1c0d37bda51c1e1be2151c560c9c973959ecR977